frontend/src/components/GenerateSchedulesUnitAndTimePicker.js

```diff
@@ -0,0 +1,184 @@
+                                value={unit}
+                                onChange={setMin(this.target.value())}
+
+                            />
+                        </form>
+                    </Col>
+
+                </Row>
+                <Row>
+                    <Col xs="4">
+                        <FormControl className={classes.formControl}>
+                            <InputLabel id="demo-simple-select-label">Avoid Meeting</InputLabel>
+                            <Select
+                                labelId="demo-simple-select-label"
+                                id="demo-simple-select"
+                                value={date}
+                                onChange={handleChange}
+                            >
+                                <MenuItem value={1}>M</MenuItem>
+                                <MenuItem value={2}>T</MenuItem>
+                                <MenuItem value={3}>TH</MenuItem>
+                                <MenuItem value={4}>W</MenuItem>
+                                <MenuItem value={5}>F</MenuItem>
+                            </Select>
+                        </FormControl>
+                        </Col>
+                    <Col xs="4">
+
+                        <form className={classes.container} noValidate>
+                            <TextField
+                                id="StartTime"
+                                label="From"
+                                type="time"
+                                defaultValue="00:00"
+                                className={classes.textField}
+                                InputLabelProps={{
+                                    shrink: true,
+                                }}
+                                inputProps={{
+                                    step: 300, // 5 min
+                                }}
+                            />
+                        </form>
+                    </Col>
+                    <Col xs="4">
+                        <form className={classes.container} noValidate>
+                            <TextField
+                                id="EndTime"
+                                label="To"
+                                type="time"
+                                defaultValue="00:00"
+                                className={classes.textField}
+                                InputLabelProps={{
+                                    shrink: true,
+                                }}
+                                inputProps={{
+                                    step: 300, // 5 min
+                                }}
+                            />
+                        </form>
+                    </Col>
+
+                </Row>
+    </Col>
+    <Col>
+    <Button
+type="submit"
+id="generatePageSearch"
+onClick={handleSubmit}
+    >Search</Button>
+    </Col>
+</Row>
+
+</Container>
+
+
+        </>
```

Comment:
this outer element isn't needed if the whole thing is wrapped in a component (row or container)